datasets/flwr_datasets/merge_resplitter.py

```diff
@@ -0,0 +1,91 @@
+"""MergeResplitter class for Flower Datasets."""
+import collections
+from typing import Dict, List, Tuple
```

Comment:
The copyright notice is missing in this file.

```suggestion
"""MergeResplitter class for Flower Datasets."""


import collections
from typing import Dict, List, Tuple
```